src/dash-adapter.js

```diff
@@ -58,7 +58,7 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
    * @private
    * @static
    */
-  static _drmProtocols: Array<Function> = [Widevine, PlayReady];
+  static _drmProtocols: Array<Function> = Env.browser.name === 'Edge' ? [PlayReady, Widevine] : [Widevine, PlayReady];
```

Comment:
@Yuvalke did you verify this actually works?
Shaka doesn’t manage priority on the config option according to docs, but according to order in manifest.

Reply:
How could I know which DRM protocol actually used?

Reply:
I tested the order and I saw it's playing regularly, i'll look for docs in Shaka

Reply:
It works